src/DependencyInjection/Configuration.php

```diff
@@ -32,6 +32,7 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->scalarNode('root_namespace')->defaultValue('App')->end()
+                ->booleanNode('is_typed')->defaultFalse()->end()
```

Comment:
I would prefer something more meaningful:
```suggestion
                ->booleanNode('typed_properties')->defaultFalse()->end()
```